cli/lsp/documents.rs

```diff
@@ -1111,6 +1182,33 @@ impl Documents {
   }
 }
 
+/// Loader that will look at the open documents.
+pub struct DocumentsDenoGraphLoader<'a> {
```

Comment:
What's the use case for this loader? Can you update the comment?

Reply:
This is part of the other PR that was merged https://github.com/denoland/deno/pull/16353

Reply:
It's a loader that will look at open documents in the lsp—could you provide a suggestion to improve the existing comment?

Reply:
`OpenDocumentsGraphLoader` might be a bit better name? Though it's a nitpick so feel free to ignore